wiki/cache.md

```
@@ -113,17 +113,17 @@ GoBricks applies production-safe cache manager defaults when cache is configured
 
 | Setting | Default | Purpose |
 |---------|---------|---------|
-| `manager.max_size` | 100 | Maximum tenant cache instances |
-| `manager.idle_ttl` | 15m | Close idle cache connections |
-| `manager.cleanup_interval` | 5m | Frequency of idle cache cleanup |
+| `manager.maxsize` | 100 | Maximum tenant cache instances |
+| `manager.idlettl` | 15m | Close idle cache connections |
+| `manager.cleanupinterval` | 5m | Frequency of idle cache cleanup |
 
 **Override defaults** in `config.yaml`:
 ```yaml
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)